sections/quick-order-form.liquid

```diff
@@ -67,13 +99,15 @@
             </thead>
 
             <tbody>
-              {%- render 'quick-order-list-row',
-                item: section.settings.collection.products.first,
-                image: section.settings.collection.products.first.featured_media,
-                sku: section.settings.collection.products.first.selected_or_first_available_variant.sku,
-                variant: section.settings.collection.products.first.selected_or_first_available_variant
+              {%-
```

Comment:
Thinking more about it and the design we have, I dont think the product row needs to use `quick-order-list-row`. Can we create a separate snippet for it? We only need the product title and image and the quick order list row loads way more information than that

Reply:
that makes sense to me 👍

Reply:
Done [here](https://github.com/Shopify/dawn/pull/3072/commits/b633c12de2d2a74e739e815663328232d5f39131)